frontend/appflowy_flutter/lib/workspace/presentation/settings/pages/account/account_deletion.dart

```diff
@@ -125,7 +129,7 @@ class _AccountDeletionDialog extends StatelessWidget {
         ),
         const VSpace(12.0),
         FlowyTextField(
-          hintText: LocaleKeys.settings_user_email.tr(),
+          hintText: _confirmText,
           controller: emailController,
```

Comment:
We can rename this `emailController` to just `controller` or similar.
```suggestion
          controller: controller,
```